### PR TITLE
button added

### LINKED ISSUE #1230: [Add] <Create a unique subscribe button with exquisite animations>
### Details

Create a unique subscribe button with exquisite animations

### Screenshots

_No response_

### Type of Contribution

- [ ] Update to an existing Animated Button
- [X] Adding a new Button
- [ ] Resolving a bug
- [ ] Proposal to the Repository
- [ ] Changes related to documentation or README.md
- [ ] Other Changes

### Checklist

- [X] I have checked the existing [issues](https://github.com/Spyware007/Animating-Buttons/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/Spyware007/Animating-Buttons/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by karthiknadar1204**
## Fixes Issue

Closes #1230 

## Checklist
Please refer pr #1448 for the proposed changes

- [x] You've linked this PR to the correct issue.
- [x] You have checked that the code is working correctly.
- [x] You ⭐️ the repository!

### CODE REVIEW

**Comment by adity2911** [comment]
Resolve conflicts ASAP and tag me

**Comment by karthiknadar1204** [comment]
Honestly someone named lakhan ya something seems to be adding someting again and again, this was te second time I had to resolve the conflict manually

**Review by Souravvmishra — CHANGES_REQUESTED** [review]
You forget to remove extra = from line number 865 in data.js

**Comment by karthiknadar1204** [comment]
@Souravvmishra Please check now

**Comment by karthiknadar1204** [comment]
Screenshot 2023-06-22 at 11 03 31 PM

Please see this snap @Souravvmishra , I can't find any = in data.js line 865

**Comment by karthiknadar1204** [comment]
Good to go ?
@Souravvmishra

File: src/Data.js
Comment (adity2911) [inline src/Data.js:1]: make sure that you only add significant data files in data.js

**Review by adity2911 — CHANGES_REQUESTED** [review]

**Comment by karthiknadar1204** [comment]
in data.js , i only added my github username

**Comment by adity2911** [comment]
I can see around 20 lines added to the file data.js. Look into it at the files changed tab.

**Comment by karthiknadar1204** [comment]
not added by me @adity2911 ,other pr's are getting merged so branch conflicts,I'll clear the present conflict,then you check again

**Pull request merged**